velox/functions/lib/SubscriptUtil.h

```diff
@@ -295,6 +363,20 @@ class SubscriptImpl : public exec::Subscript {
     return rawOffsets[indices[row]] + index;
   }
 
+  void addError(
+      vector_size_t row,
+      std::exception_ptr newError,
+      std::exception_ptr& error,
+      SelectivityVector& errorRows) const {
+    if (!error) {
+      error = newError;
+    }
+    if (errorRows.size() <= row) {
+      errorRows.resize(row + 1, false);
+    }
+    errorRows.setValid(row, true);
```

Comment:
Can record the errors for each failed row? Have a chat with Wei and some Meta internal query system might need these distinct error type report for monitoring purpose. Thanks!